src/components/Photo/Selfie.js

```diff
@@ -73,8 +73,15 @@ class SelfieCapture extends Component<Props, State> {
     }))
   }
 
-  takeSnapshot = () =>
+  takeSnapshot = () => {
+    const snapshot =
+      this.state.snapshotBuffer[0] || this.state.snapshotBuffer[1]
+
+    snapshot?.blob &&
+      this.state.isCaptureButtonDisabled &&
```

Comment:
Think this `&&` shouldn't be here since the next line is changing the state of `isCaptureButtonDisabled`.

Reply:
We only want to set `isCaptureButtonDisabled` to `false` if `isCaptureButtonDisabled` is currently `true`. So it is needed, we don't want to set it to false if it's already false.

Reply:
Ah, ok. Had a feeling that was the case. Personally, I feel an IF statement would've been clearer here then.